src/components/navbar/navbar.tsx

```diff
@@ -1,23 +1,33 @@
-import { Link, useNavigate } from "react-router-dom";
+import { NavLink, useNavigate } from "react-router-dom";
 import React from "react";
 import { useSelector, useDispatch } from "react-redux";
 import { RootState } from "../../store/store";
 import { logoutAction } from "../../store/slice/authSlice";
+import "./navbar.css";
+import { Button } from "antd/lib/radio";
 
 export const Navbar: React.FC = () => {
   const dispatch = useDispatch();
-  const isLoggedIn = useSelector((state: RootState) => state.isLoggedIn.isAuth);
+  // const isLoggedIn = useSelector((state: RootState) => state.isLoggedIn.isAuth);
   const navigation = useNavigate();
+
   const handleLogout = () => {
     dispatch(logoutAction());
     localStorage.setItem("isLoggedIn", JSON.stringify(false));
     navigation({ pathname: "/login" });
```

Comment:
why done be have the logic of cleaning up the localstory inside logoutaction ?

Reply:
Is it required to clean local storage inside the logout action? I have changed the local storage variable to false inside logout action.